test/CMakeLists.txt

```diff
@@ -369,18 +380,19 @@ endif()
 if(ENABLE_PYTHON_BINDINGS AND ENABLE_DATA_TOOLS)
   find_package(Python COMPONENTS Interpreter)
   add_custom_command(OUTPUT python_valhalla.log
-    COMMAND
-      LOCPATH=${VALHALLA_SOURCE_DIR}/locales
-      PYTHONPATH=${CMAKE_BINARY_DIR}/src/bindings/python
-      ASAN_OPTIONS=verify_asan_link_order=0:detect_leaks=0
-      /bin/bash -cx "${Python_EXECUTABLE} -m unittest discover -s ${VALHALLA_SOURCE_DIR}/test/bindings/python -v > ${CMAKE_BINARY_DIR}/test/python_valhalla.log 2>&1 || cat ${CMAKE_BINARY_DIR}/test/python_valhalla.log"
-    WORKING_DIRECTORY ${CMAKE_BINARY_DIR}
-    DEPENDS
-      ${VALHALLA_SOURCE_DIR}/test/bindings/python/test_utrecht.py
-      ${VALHALLA_SOURCE_DIR}/test/bindings/python/valhalla.json
-      utrecht_tiles
-      python_valhalla
-    VERBATIM)
+  COMMAND ${CMAKE_COMMAND} -E env
+    LOCPATH=${VALHALLA_SOURCE_DIR}/locales
+    PYTHONPATH=${CMAKE_BINARY_DIR}/src/bindings/python
+    ASAN_OPTIONS=verify_asan_link_order=0:detect_leaks=0
+    ${Python_EXECUTABLE} -m unittest discover -s ${VALHALLA_SOURCE_DIR}/test/bindings/python -v
```

Comment:
This is replacing a bash command with CMake command. Since I still have errors while building the solution I've not been able to verify this yet.